src/DgmOctree.cpp

```diff
@@ -3502,14 +3509,20 @@ unsigned DgmOctree::executeFunctionForAllCellsAtLevel(	unsigned char level,
 		s_jumps = 0.0;
 		s_binarySearchCount = 0.0;
 #endif
-
+#ifdef CC_CORE_LIB_USES_QT_CONCURRENT
+		// The QtConcurrent is used in case both Qt and TBB are available
```

Comment:
Isn't it only if Qt is available?

(and I would remove 'The' in front of QtConcurrent)